src/patternfly/layouts/Page/page.scss

```diff
@@ -155,38 +137,21 @@ $nav: "pf-c-nav";
   }
 
   .pf-l-page__header-tools {
-    .pf-l-toolbar {
-      margin-top: var(--pf-l-page__header-tools--toolbar--MarginTop);
-      margin-bottom: var(--pf-l-page__header-tools--toolbar--MarginBottom);
-    }
-  }
-
-  .pf-c-nav__horizontal-list .pf-c-nav__link {
-    padding-top: var(--pf-c-nav__horizontal-list-link--PaddingTop);
-    padding-bottom: var(--pf-c-nav__horizontal-list-link--PaddingBottom);
+    margin-top: var(--pf-l-page__header-tools--MarginTop);
+    margin-bottom: var(--pf-l-page__header-tools--MarginBottom);
   }
 
-  .pf-l-page__header-selector {
-    padding-top: var(--pf-l-page__header-selector--PaddingTop);
-    padding-bottom: var(--pf-l-page__header-selector--PaddingBottom);
-  }
-
-  &.pf-m-expanded {
-    min-height: var(--pf-l-page__header--m-expanded--MinHeight);
+  &.pf-m-tall {
+    min-height: var(--pf-l-page__header--m-tall--MinHeight);
 
     .pf-l-page__header-brand {
-      padding-top: var(--pf-l-page__header-brand--m-expanded--PaddingTop);
-      padding-bottom: var(--pf-l-page__header-brand--m-expanded--PaddingBottom);
-    }
-
-    .pf-l-toolbar {
-      margin-top: var(--pf-l-page__header-tools--toolbar--m-expanded--MarginTop); // include margin-top for avatar to position itself vertically
-      margin-bottom: var(--pf-l-page__header-tools--toolbar--m-expanded--MarginBottom);
+      padding-top: var(--pf-l-page__header-brand--m-tall--PaddingTop);
+      padding-bottom: var(--pf-l-page__header-brand--m-tall--PaddingBottom);
     }
 
-    .pf-c-nav__horizontal-list .pf-c-nav__link {
-      padding-top: var(--pf-c-nav__horizontal-list-link--m-expanded--PaddingTop);
-      padding-bottom: var(--pf-c-nav__horizontal-list-link--m-expanded--PaddingBottom);
+    .pf-l-page__header-tools {
+      margin-top: var(--pf-l-page__header-tools--m-tall--MarginTop); // include margin-top for avatar to position itself vertically
+      margin-bottom: var(--pf-l-page__header-tools--m-tall--MarginBottom);
```

Comment:
I guess this isn't actually new, but doesn't this break our rule of pad yourself but only margin your children?